### PR TITLE
Rework opx_build_all script to allow single component build

### PR DESCRIPTION
**Pull request by pkarashchenko**

### CODE REVIEW

**Comment by pkarashchenko** [comment]
@johntconklin Please check. I have updated to use default to "all" if no parameters are specified. Also now it should be possible to build few components in a row.

**Comment by pkarashchenko** [comment]
I have updated README.md to reflect latest changes. Please review.

File: README.md [hunk 1]
```diff
@@ -44,17 +44,18 @@ To setup your Docker OPX image, use the script in the _opx-build/scripts_ folder
...
     cd ~/dev/opx
-    ./opx-build/scripts/opx_run
+    opx-build/scripts/opx_run
 
 To build a single repository, go to the repository and build. For example, to build `opx-logging`:
 
-    opx-dev@077f7b30f8ef:/# cd /mnt/opx-logging
-    opx-dev@077f7b30f8ef:/mnt/opx-logging# git-buildpackage --git-dist=jessie --git-ignore-branch --git-pbuilder
+    opx-dev@077f7b30f8ef:/# cd /mnt
+    opx-dev@077f7b30f8ef:/mnt# opx-build/scripts/opx_build opx-logging
```
Comment (jghalam) [inline README.md]: shouldn't it be: opx-dev@077f7b30f8ef:/mnt# opx-build/scripts/opx_build_all opx-logging

File: scripts/opx_build [hunk 2]
```diff
@@ -53,6 +53,10 @@ repos=( opx-logging
 	opx-tmpctl
 	)
 
+if [ ! -z "$1" ] && [ "$1" != "all" ]; then
```
Comment (jghalam) [inline scripts/opx_build]: do we really need the keyword "all"?

File: README.md [hunk 3]
```diff
@@ -44,17 +44,18 @@ To setup your Docker OPX image, use the script in the _opx-build/scripts_ folder
...
 
 ## Build all repositories
-Issue the `opx_build_all` command from the root directory to build all repos and create packages in the same root directory.
+Issue the `opx_build all` command from the root directory to build all repos and create packages in the same root directory.
 
-    opx_build_all
+    opx-dev@077f7b30f8ef:/# cd /mnt
+    opx-dev@077f7b30f8ef:/mnt# opx-build/scripts/opx_build all
```
Comment (jghalam) [inline README.md]: opx_build should be opx_build_all

File: README.md (hunk 1 above)
Comment (pkarashchenko) [inline README.md]: The script is renamed in this PR: "opx_build_all" -> "opx_build"

File: scripts/opx_build (hunk 2 above)
Comment (pkarashchenko) [inline scripts/opx_build]: The script is renamed in this PR: "opx_build_all" -> "opx_build"

Should I revert to original name?

File: README.md (hunk 3 above)
Comment (pkarashchenko) [inline README.md]: The script is renamed in this PR: "opx_build_all" -> "opx_build"

File: scripts/opx_build (hunk 2 above)
Comment (jghalam) [inline scripts/opx_build]: Now I see that you have renamed the script to opx_build. Please ignore the comments about script name. But I think the "all" is still unnecessary.

Comment (pkarashchenko) [inline scripts/opx_build]: Agree, not needed. Just done to be similar with "make all". Can be removed.
@johntconklin What is your suggestion here?

Comment (jghalam) [inline scripts/opx_build]: I'm ok with this the way it is.

**Pull request merged**